src/test/testAssign.js

```diff
@@ -0,0 +1,130 @@
+            .set("Authorization", userMock.token.operator)
+            .send({"email": "happyka52@gmail.com"})
+            .end((err, res) => {
+                expect(res.statusCode).to.equal(200);
+                done();
+            });
+    });
+
+    it('it should not assign driver if driver is already assigned', (done) => {
+    
+        chai.request(app)
+            .patch('/api/v1/buses/assigndriver/' + busId)
+            .set("Authorization", userMock.token.operator)
+            .send({"email": "happyka52@gmail.com"})
+            .end((err, res) => {
+                expect(res.statusCode).to.equal(409);
+                done();
+            });
+    });
+    //karigirwa@gmail.com
+    
+    it('it should not assign driver if bus is already assigned', (done) => {
+    let Id=1
+        chai.request(app)
+            .patch('/api/v1/buses/assigndriver/' + Id)
+            .set("Authorization", userMock.token.operator)
+            .send({"email": "karigirwa@gmail.com"})
+            .end((err, res) => {
+                expect(res.statusCode).to.equal(409);
+                done();
+            });
+    });
+
+    it('it should not unassign driver if driver is not assigned', (done) => {
+        let Id=1
+            chai.request(app)
+                .patch('/api/v1/buses/unassigndriver/' + Id)
+                .set("Authorization", userMock.token.operator)
+                .send({"email": "karigirwa@gmail.com"})
+                .end((err, res) => {
+                    expect(res.statusCode).to.equal(401);
+                    done();
+                });
+        });
+
+        it('it should unassign driver', (done) => {
+            let Id=1
+                chai.request(app)
+                    .patch('/api/v1/buses/unassigndriver/' + Id)
+                    .set("Authorization", userMock.token.operator)
+                    .send({"email": "happyka52@gmail.com"})
+                    .end((err, res) => {
+                        expect(res.statusCode).to.equal(200);
+                        done();
+                    });
+            });
+
+            it('it should not retrieve assigned buses when page not found', (done) => {
+                chai.request(app)
+                    .get('/api/v1/buses/assignedbuses?page=4&limit=10')
+                    .set("Authorization", userMock.token.operator)
+                    .end((err, res) => {
+                        expect(res.statusCode).to.equal(404);
+                        done();
+                    });
+            });
+
+
+            it('it should not retrieve assigned buses when limit is undefined', (done) => {
+                chai.request(app)
+                    .get('/api/v1/buses/assignedbuses?page=1&limit=undefined')
+                    .set("Authorization", userMock.token.operator)
+                    .end((err, res) => {
+                        expect(res.statusCode).to.equal(500);
+                        done();
```

Comment:
You should not only `expect` for statusCode, you should check for other attributes for example the return `message`